meerkat_analysis/indicators.py

```diff
@@ -41,13 +41,23 @@ def fix_dates(start_date, end_date, epi_week_start_day):
     return start_date, end_date, freq
 
 
-def count(data, var_id, start_date=None, end_date=None, epi_week_start_day=None):
+def calculate(data, var_id,calc,start_date=None, end_date=None, epi_week_start_day=None):
     """
     We return the total count of var_id and a timeline by epi_week
 
     Args:
         data: data in dataframe
         var_id: the variable id to count
+        calc: the mathematical function to calculate
+        ot: overall total
```

Comment:
I think you want to indent all the possible values of the variable `calc`? Just so it is clear that they are possible values of "calc" and not extra arguments of the function calculate.